packages/graphql/src/schema/microLearning.ts

```diff
@@ -2,16 +2,18 @@ import * as DB from '@klicker-uzh/prisma'
 import builder from '../builder'
 import type { ICourse } from './course'
 import { CourseRef } from './course'
-import { ElementStackRef, IElementStack } from './practiceQuizzes'
+import {
+  ElementStackRef,
+  IElementStack,
+  PublicationStatus,
+} from './practiceQuizzes'
 
 export interface IMicroLearning extends DB.MicroLearning {
   course?: ICourse | null
   stacks?: IElementStack[]
-}
 
-export const MicroLearningStatus = builder.enumType('MicroLearningStatus', {
```

Comment:
This was required for consistency with the database. There, the microlearning has a separate status, which does not correspond to the publication status